internals/webhooks/defaults_webhook_test.go

```diff
@@ -0,0 +1,56 @@
+package webhooks
+
+import (
+	"testing"
+
+	rcsv1alpha1 "github.com/dana-team/container-app-operator/api/v1alpha1"
+	. "github.com/onsi/gomega"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	knativev1 "knative.dev/serving/pkg/apis/serving/v1"
+)
+
+type TestCase struct {
+	cappName    string
+	scaleMetric string
+	mutated     bool
+}
+
+func TestDefaultsWebhook(t *testing.T) {
+	tests := []TestCase{
+		{
+			cappName: "i-forgot-metric",
```

Comment:
change the test name to something like `withoutScaleMetric` and `withScaleMetric`